specification/src/main/asciidoc/platform/Security.adoc

```diff
@@ -108,7 +108,7 @@ each role to determine if it can map the user to the role.
 _link:#a261[See URL Authorization]_
 shows this process.
 
-image:Platform_Spec-5.png[image]
+image:images/Platform_Spec-5.png[image]
```

Comment:
There seems to be some disagreement over how to specify the images.
This all worked for me without any path information in the image references,
for both html and pdf.  I don't understand why "images/" needs to be added
here.  And I've seen "../images/" added in other sections.

Someone needs to explain why it doesn't work as is, then explain how to
update it consistently in all sections.

Reply:
@bshannon All of the image processing still needs work.  Let's not hold up this review on this global item.  This section is getting too far out of date with regards to the other cleanup.  Can we approve this PR and log an issue for images?  It's on my list of TODOs, but an issue would be even better reminder.  Thanks.

Reply:
I guess I would prefer that this just be put back to the way it was
and then if there really is something broken they can all be fixed together.
But I won't hold up the review for this.